build_packages

```diff
@@ -301,6 +301,11 @@ test_image_content "${BOARD_ROOT}"
 # upload packages if enabled
 upload_packages
 
+# build and label the system before generating torcx image.
+info "Label the build root"
+sudo chroot "${BOARD_ROOT}" bash -c "cd /usr/share/selinux/mcs && semodule -s mcs -i *.pp"
+sudo setfiles -Dv -r "${BOARD_ROOT}" "${BOARD_ROOT}"/etc/selinux/mcs/contexts/files/file_contexts "${BOARD_ROOT}"
```

Comment:
`setfiles` is a cool idea, but doing it in `build_packages` is not useful, because the images are not built from the files in `BOARD_ROOT`. This would have to happen somewhere in `build_image` and in `build_torcx_store`.

Reply:
Also came across this: https://github.com/gardenlinux/gardenlinux/blob/main/bin/makepart#L20-L22